yeahcheese/app/Http/Controllers/EventController.php

```diff
@@ -60,15 +60,17 @@ public function store(Request $request)
     public function show(Request $request)
     {
         $event = Event::where('auth_key', $request->auth_key)->first();
-        // 今の段階で画像を登録する機能がないので画像は投げずにイベントのみ受け渡し
-        // 画像の有無を確認する機能が必要
-        //$picture = Picture::where('event_id', $event->id)->get();
-        //$data = [['pictures' => $picture], ['event' => $event]];
-        return view('event', ['event' => $event]);
+        if(is_null($event))
```

Comment:
null判定より exists() 使った方がよさそう
```suggestion
        if($event->exists())
```

Reply:
ありがとうございます。それで書き直してみます

Reply:
existsだと手元の環境だと動かなかったのでis_nullのままにします

Reply:
first()してたので
`Event::where('auth_key', $request->auth_key)->exists()`
でした。すみません。

参考: https://readouble.com/laravel/7.x/ja/queries.html#aggregates

Reply:
ありがとうございます。